### PR TITLE
QA Observation Bugfix/FOUR-13236: Error in console when clicking on a process box

### PR DESCRIPTION
**Pull request by CarliPinell**
## Solution
- Reported console error was fixed

## How to Test
- Go to Process Catalogue
- Click on some Available Process Category
- Click on Any Process
- Check browser console

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-13236

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.
ci:deploy
ci:next

### CODE REVIEW

File: ProcessMaker/PackageHelper.php [hunk 1]
```diff
@@ -50,6 +50,8 @@ class PackageHelper
 
     const PM_PACKAGE_DECISION_ENGINE = 'ProcessMaker\Package\PackageDecisionEngine\PackageServiceProvider';
 
+    const PM_PACKAGE_SAVEDSEARCH = 'ProcessMaker\Package\SavedSearch\SavedSearchServiceProvider';
```
Comment (fagubla) [inline ProcessMaker/PackageHelper.php:53]: PM_PACKAGE_SAVED_SEARCH

**Review by fagubla — CHANGES_REQUESTED** [review]

File: ProcessMaker/PackageHelper.php (hunk 1 above)
Comment (CarliPinell) [reply ProcessMaker/PackageHelper.php]: Solved

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-1bc643fd3c.engk8s.processmaker.net

**Comment by pmPaulis** [comment]
@CarliPinell 
The issue is still replicating in the build generated
image

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-1bc643fd3c.engk8s.processmaker.net

**Review by fagubla — APPROVED** [review]

**Comment by pmPaulis** [comment]
@CarliPinell 
The issue does not replicate but the following use case is not working in Open Modeler:

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-1bc643fd3c.engk8s.processmaker.net

**Comment by pmPaulis** [comment]
@CarliPinell 
It works as expected
image
image

**Review by pmPaulis — APPROVED** [review]

**Pull request merged**